platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/crypto/CryptoStatic.java

```diff
@@ -232,6 +235,22 @@ public static X509Certificate generateCertificate(
         }
     }
 
+    /**
+     * Generate a X509Certificate from a byte array that was previously obtained via X509Certificate.getEncoded().
+     *
+     * @param encoded a byte array with an encoded representation of a certificate
+     * @return the certificate reconstructed from its encoded form
+     */
+    @NonNull
+    public static X509Certificate generateCertificate(@NonNull final byte[] encoded) {
```

Comment:
Is this really generating a new certificate or decoding it from the encoding?    I think the term `generate` in cryptography means to make something new.

Reply:
Good point. The method is now renamed to `decodeCertificate`.